qmcpy/test/test_integrate.py

```diff
@@ -1,44 +1,48 @@
+import numpy as np
 import unittest
 
-from numpy import arange
-
-from algorithms.distribution import measure
+from algorithms.distribution import Measure
 from algorithms.distribution.IIDDistribution import IIDDistribution
 from algorithms.distribution.QuasiRandom import QuasiRandom
-from algorithms.function.KeisterFun import KeisterFun
-from algorithms.function.AsianCallFun import AsianCallFun
+from algorithms.function.keister_integrand import KeisterFun
+from algorithms.function.asian_call_integrand import AsianCallFun
 from algorithms.stop.CLTStopping import CLTStopping
-from algorithms.stop.CLT_Rep import CLT_Rep
+from algorithms.stop.CLTRep import CLTRep
 from algorithms.integrate import integrate
 
+
 class IntegrationExampleTest(unittest.TestCase):
     '''
     def test_qmcpy_version(self):
         import qmcpy
         self.assertEqual(qmcpy.__version__, 0.1)
     '''
 
-    def test_KeisterFun(self):
-        absTol = .3
+    def test_keister_integrand(self):
+        abs_tol = .3
         dim = 2
-        fun = KeisterFun()
-        measureObj = measure().IIDZMeanGaussian(dimension=[dim],variance=[1 / 2])
-        distribObj = IIDDistribution(trueD=measure().stdGaussian(dimension=[dim]))
-        stopObj = CLTStopping(distribObj,nInit=16, absTol=absTol, alpha=.01, inflate=1.2)
-        sol, out = integrate(fun, measureObj, distribObj, stopObj)
+        integrand = KeisterFun()
+        measure = Measure().iid_zmean_gaussian(dimension=[dim], variance=[.5])
```

Comment:
Eventually, I think it would be better to have a `classmethod` in `Measure` which can create these objects ... I find the ```measure = Measure().iid_zmean_gaussian(dimension=[dim], variance=[.5])``` line kinda gross.

Reply:
updated in most recent commit
Measure in now an abstract class that has instances in algorithms/distributuions/Measures.py

New lines reads:
    from algorithms.distributions.Measures import IIDZeroMeanGaussian
    measure = IIDZeroMeanGaussian(...)